tenseal/cpp/tensors/utils/utils.h

```diff
@@ -93,7 +93,7 @@ shared_ptr<T> pack_vectors(const vector<shared_ptr<T>>& vectors) {
     replicate_vector(replicated_mask, slot_count);
 
     auto packed_vec = vectors[0]->copy();
-    packed_vec->size(slot_count);
+    packed_vec->chunked_size({slot_count});
```

Comment:
This function can actually be redesigned to support multi-ciphertext vectors. Not necessarily in this PR.